src/daemon/config.rs

```diff
@@ -61,6 +61,11 @@ impl ConfigDefaults {
     fn use_ta() -> bool {
         env::var(KRILL_ENV_USE_TA).is_ok()
     }
+
+    fn ta_aia() -> uri::Rsync {
+        uri::Rsync::from_str("rsync://localhost/ta/ta.cer").unwrap()
```

Comment:
Is localhost the correct default here? Or should it use the `service_uri`?

Reply:
Ah it depends on the publisher URI which it doesn't know until it is configured with the command line tools.